.github/workflows/ci.yaml

```diff
@@ -270,4 +270,98 @@ jobs:
         curl -s https://codecov.io/bash | bash
         echo "Uploaded"
 
+  linux-fpga:
+    continue-on-error: true
```

Comment:
This flag is important since it means even if the steps in this job fails, you won't block the build. Remove this when ready to start requiring CI checks for your builds.